src/scores.js

```diff
@@ -1,14 +1,43 @@
+const postScore = async (newScore) => {
+  const sendInfo = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/nUT3TlYnotX0jWAN8AyD/scores/', {
+    method: 'POST',
+    body: JSON.stringify({
+      user: newScore.user,
+      score: newScore.score,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const json = await sendInfo.json();
+  return json;
+};
+
+const getScore = async () => {
+  const getInfo = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/nUT3TlYnotX0jWAN8AyD/scores/');
+  const json = await getInfo.json();
+  return json;
+};
```

Comment:
New data is properly being submitted to the API and after the refresh button is clicked, new info is rendered as expected 🙌👌 Fantastic job!!

Reply:
# Thanks!

<img src="https://media1.giphy.com/media/dYdrcYcidefPzFSYJd/giphy.gif"/>